### PR TITLE
Improved some descriptions of the task

### PR DESCRIPTION
**Pull request by ArcEglos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,32 +18,44 @@
...
+There is a basic search feature in our app that allows search for patients based on several properties. The implementation of this feature has some problems. We're interested in two of them:
 
-It's a little janky, sometimes it doesn't search for the correct query. The component boundary is confusing also.
+1. There is a bug leading to incorrect results in some cirumstances. One simple way to cause it is typing "J" into the search after refreshing the page. It should show two search results, but nothing is displayed.
+2. The boundaries between the components in the implementation is convoluted.
 
 ### 1.1 Add a test
 
-Add a test for this component. Feel free to use whichever testing library you think is best.
+Add a test for the search, it should cover the bug described above and any other testcases you feel are important. Feel free to use whichever testing library you think is best.
```
Comment (protiumx) [inline README.md:28]: Maybe we want to rephrase the last sentence to `feel free to use whichever react testing library along with jest`

File: README.md [hunk 2]
```diff
@@ -18,32 +18,44 @@
...
+### 1.2 Refactor
 
-Now that we have a useful test for this component. Please refactor this component to remove the peculiar behaviour when searching for something. Bonus points for addressing the confusing component boundary.
+Now that we have a useful test for this component: Please refactor this component to remove the problems described above (the search bug as well as the convoluted component boundary).
 
-Add a loading spinner (feel free to use anything) to provide the user with some visual feedback while it is waiting for a network request to complete.
+### 1.3 Add a feature
 
-## Task 2 
+Add a loading spinner (feel free to use anything) to provide the user with some visual feedback while the code is waiting for a network request to complete.
```
Comment (protiumx) [inline README.md]: This `(feel free to use anything)` feels redundant

File: src/patients/patients_search.tsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
-import { FunctionComponent, useState } from "react";
-import { Patient, PatientSearchQuery } from "./patients";
+import { FunctionComponent, useState } from 'react';
+import { Patient, PatientSearchQuery } from './patients';
```
Comment (protiumx) [inline src/patients/patients_search.tsx:2]: This seems to be formatted this way only in this file. Could you apply the same format to old the files?

File: README.md (hunk 1 above)
Comment (ArcEglos) [reply README.md:28]: I think if someone would actually rip out jest that would also be an interesting signal. I would at least want to know the reasoning for it

**Review by protiumx — APPROVED** [review]
🚀

**Pull request merged**